### ISSUE TITLE
warnings during installation

### ISSUE THREAD

**Issue by szabgab** [tadzik/Template-Mojo#31]
```
WARNINGS for /home/ubuntu/.zef/store/Template-Mojo.git/3d38f92636a0c44dfd4cec2e7b8b67f12e43c026/EVAL_7:
Useless use of constant string "blarg" in sink context (line 1)
WARNINGS for /home/ubuntu/.zef/store/Template-Mojo.git/3d38f92636a0c44dfd4cec2e7b8b67f12e43c026/EVAL_13:
Useless use of constant integer 0 in sink context (line 1)
```

Using Rakudo 2017.06 on Linux.

**Comment by tadzik** [comment]
Indeed, the test cases in line 16 and 22 are to blame; they also seem to be testing if indeed, a leftover expression value doesn't end up in the output, so it is indeed a useless value in sink context. Not sure how to fix that without making the tests a bit less thorough though.

**Comment by JJ** [comment]
This template is converted in this code:

```raku
sub anon { my $_M = "";;$_M ~= 'hello '; "blarg" ;$_M ~= ' world';;return $_M;}
```

As you can see, the constant is there, by itself, in a line. Probably the parsing is to blame, it should be turned into... not really clear what. Possibly $_M ~= "blarg". Not clear why that's not the case.

**Comment by JJ** [comment]
This probably calls for an independent grammar module, with independent testing. It's not something that can be fixed easily.
